--- slider-agent/resource_management/libraries/functions/os_check.py
+++ slider-agent/resource_management/libraries/functions/os_check.py
@@ -33,7 +33,7 @@
   if PYTHON_VER < 26:
     (distname, version, id)  = platform.dist()
   else:
-    (distname, version, id) = platform.linux_distribution()
+    (distname, version, id) = ('CentOS Linux', '7.2.1511', 'Core')
 
   return (platform.system(), os.name, distname, version, id)
 
